microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/graphql/java/MicroProfileGraphQLASTValidator.java

```diff
@@ -44,6 +46,7 @@ public class MicroProfileGraphQLASTValidator extends JavaASTValidator {
 	private static final Logger LOGGER = Logger.getLogger(MicroProfileGraphQLASTValidator.class.getName());
 
 	private static final String NO_VOID_MESSAGE = "Methods annotated with microprofile-graphql''s `@Query` cannot have ''void'' as a return type.";
+	private static final String GRAPH_QL_API_ANNOTATION_MESSAGE = "In order for microprofile-graphql to recognize ''{1}'' as a part of your GraphQL API, you need to annotate ''{0}'' with ''@GraphQLApi''.";
```

Comment:
A suggestion about the message just to avoid using personal pronouns (I'm not sure if there is a standard?): 
`Annotate ''{0}'' with ''@GraphQLApi'' in order for microprofile-graphql to recognize ''{1}'' as a part of the GraphQL API.`